lib/ansible/modules/network/cloudengine/ce_ntp.py

```diff
@@ -442,7 +442,7 @@ def get_ntp_exist_config(self):
 
         # get all ntp config info
         root = ElementTree.fromstring(xml_str)
-        ntpsite = root.findall("data/ntp/ntpUCastCfgs/ntpUCastCfg")
```

Comment:
Finding all elements from root node of which is named 'data' is failed.
It is right to remove the 'data' tag to find all nodes from xml data.